peachjam/admin.py

```diff
@@ -251,6 +258,7 @@ class DocumentAdmin(admin.ModelAdmin):
                     "title",
                     "date",
                     "language",
+                    "allow_robots",
```

Comment:
Hmm, let's put it under the Advanced section -- otherwise editors will wonder about it and click it randomly :)

It's not a common option, no need to fill up the key details section with something the editors won't worry about 99% of the time.